Sources/App/Controllers/PackageCollectionController.swift

```diff
@@ -0,0 +1,17 @@
+import Vapor
+
+
+struct PackageCollectionController {
+    func generate(req: Request) throws -> EventLoopFuture<PackageCollection> {
+        guard let owner = req.parameters.get("owner") else {
+            return req.eventLoop.future(error: Abort(.notFound))
+        }
+
+        return PackageCollection.generate(
+            db: req.db,
+            owner: owner,
+            authorName: "\(owner) via Swift Package Index",
+            collectionName: "\(owner) package collection"
```

Comment:
I've picked something semi-sensible here, better suggestions welcome. All we have is the owner name. Perhaps `\(owner)'s package collection`, although that can be weird / grammatically incorrect. OTOH, I'm not sure where/how/if `collectionName` would even be used downstream.

Reply:
I don't think it needs the words package collection in it. I'd be inclined to do `"Packages from \(owner)"`, `"Packages by \(owner)"`, or `"Packages authored by \(owner)"`